planner/core/util.go

```diff
@@ -231,3 +233,15 @@ func GetStatsInfo(i interface{}) map[string]uint64 {
 	statsInfos = CollectPlanStatsVersion(physicalPlan, statsInfos)
 	return statsInfos
 }
+
+// ExtractStringFromStringSet helps extract string info from set.StringSet
+func ExtractStringFromStringSet(set set.StringSet) string {
+	r := ""
+	if len(set) < 1 {
+		return r
+	}
+	for k := range set {
+		r = fmt.Sprintf("%s,%s", r, k)
```

Comment:
How about using ` bytes.Buffer` instead of `fmt.Sprintf`?

Reply:
updated.